requirements-freqai-rl.txt

```diff
@@ -0,0 +1,9 @@
+# Include all requirements to run the bot.
+-r requirements-freqai.txt
+
+# Required for freqai-rl
+torch==1.12.1
+stable-baselines3==1.6.1
+gym==0.21
+sb3-contrib==1.6.1
+tensorboard==2.10.1
```

Comment:
maybe i missunderstood your comment in https://github.com/freqtrade/freqtrade/pull/7289#discussion_r988183523 - but i think you intended to remove tensorboard (?)

Reply:
No we added it back because the automatic tensorboard integration depends on this.